### ISSUE TITLE
Constant variables from winbase.h are not documented

### ISSUE THREAD

**Issue by Alex-PLACET** [microsoft/win32metadata#189]
I try to use FormatMessageA. It's documentation is well generated but the constant flags to use with are not. Like :
WinBase.h l.2471
```
#define FORMAT_MESSAGE_IGNORE_INSERTS  0x00000200
#define FORMAT_MESSAGE_FROM_STRING     0x00000400
#define FORMAT_MESSAGE_FROM_HMODULE    0x00000800
#define FORMAT_MESSAGE_FROM_SYSTEM     0x00001000
#define FORMAT_MESSAGE_ARGUMENT_ARRAY  0x00002000
#define FORMAT_MESSAGE_MAX_WIDTH_MASK  0x000000FF
```

**Comment by kennykerr** [comment]
Transferring this issue to the win32metadata repo as these constants are missing because they are not present in metadata. They should be added.

**Comment by mikebattista** [comment]
Thanks for filing. We do have some work to do to round out our enum support and will ensure this is addressed.

**Comment by sotteson1** [comment]
Fixed with 10.0.19041.5-preview.5.
